demos/src/app_setup_options.rs

```diff
@@ -25,8 +29,12 @@ impl AppSetupConfiguration {
         Self {
             schedule_to_use: if let Some(value) = url_params.get("schedule") {
                 ScheduleToUse::from_str(&value, true).unwrap()
-            } else {
+            } else if cfg!(feature = "avian") {
```

Comment:
This will break the build for Rapier. Unlike `#[cfg]`, which prevents the compilation of the relevant thing altogether, `cfg!` merely expands to a boolean - which means that even when the `avian` feature is turned off the compiler is still going to attempt to resolve `ScheduleToUse::PhysicsSchedule` - and fail.

Reply:
[`e4d1d3b` (#61)](https://github.com/idanarye/bevy-tnua/pull/61/commits/e4d1d3bf7e5565395ac54a639145205f938e2026)